src/usb-device.adb

```diff
@@ -313,6 +313,9 @@ package body USB.Device is
    function Initialize
      (This            : in out USB_Device_Stack;
       Controller      : not null Any_USB_Device_Controller;
+      Vendor_Id       : UInt16 := 16#6666#;
```

Comment:
It would be better to add these arguments at the end, after Max_Packet_Size. This way it will not brake existing code and we can make a minor release.

Reply:
I also thought about moving all configuration into alire.toml. Since the USB.Device is a singleton on basically all systems anyway it avoids a bit of the boilerplate code.

Reply:
I don't think this belongs in crate configuration. Even if it is unlikely, I can imagine a system with two USB device ports.
Also we may want to add USB Host support in the future and the settings won't make sens for that.